wlm/src/main/java/ai/djl/serving/wlm/ModelInfo.java

```diff
@@ -964,6 +968,51 @@ void downloadS3() throws ModelException, IOException {
         }
     }
 
+    void build_trt_llm_engine() throws ModelException, IOException {
+        logger.info("Start building TensorRT-LLM engine");
+        final String trt_llm_model_repo = "/tmp/tensorrtllm";
+        prop.putIfAbsent("option.trt_llm_model_repo", trt_llm_model_repo);
```

Comment:
If we decided this location as a fixed location, you don't need to overwrite it. Just hardcode to cover in the toolkit code base

Reply:
The idea here is to not overwrite the property but rather setting this property and guide the toolkit to create model repo in this path. The same property is again passed to the toolkit in `init_inference` call. The same can be achieved by hardcoding the path in the toolkit except that this makes it easier to change in case we want to make it configurable in the future by just making the change here.